beaker-docker.gemspec

```diff
@@ -1,39 +1,40 @@
-# -*- encoding: utf-8 -*-
-$LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
+# frozen_string_literal: true
+
+$LOAD_PATH.unshift File.expand_path('lib', __dir__)
 require 'beaker-docker/version'
 
 Gem::Specification.new do |s|
-  s.name        = "beaker-docker"
+  s.name        = 'beaker-docker'
```

Comment:
I would disable `Style/StringLiterals`. There used to be a performance difference, but AFAIK in current rubies there isn't so I'd keep it simple and allow both styles.